app/models/agents/webhook_agent.rb

```diff
@@ -20,6 +20,9 @@ class WebhookAgent < Agent
         * `payload_path` - JSONPath of the attribute in the POST body to be
           used as the Event payload.  If `payload_path` points to an array,
           Events will be created for each element.
+        * `verbs` - Comma-separated list of http verbs your agent will accept.
```

Comment:
I'd suggest picking one format.   Commas is probably better?

On Monday, August 31, 2015, Darren Cauthon <notifications@github.com> wrote:

> In app/models/agents/webhook_agent.rb
> <https://github.com/cantino/huginn/pull/999#discussion_r38340797>:
>
> > @@ -20,6 +20,9 @@ class WebhookAgent < Agent
> >          * `payload_path` - JSONPath of the attribute in the POST body to be
> >            used as the Event payload.  If `payload_path` points to an array,
> >            Events will be created for each element.
> > +        * `verbs` - Comma-separated list of http verbs your agent will accept.
>
> Both! I wrote tests to cover both cases, as well as white space and
> what-not.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cantino/huginn/pull/999/files#r38340797>.
>